.appveyor.yml

```diff
@@ -32,7 +32,6 @@ cache:
 
 init:
     - SET PATH=c:\php\%PHP_VERSION%;%PATH%
-    - SET SYMFONY_DEPRECATIONS_HELPER=strict
```

Comment:
Do we need to take care of some deprecations?

Reply:
We have some deprecations in the codebase, but the reason this got removed is that it was useless as we have a good default in the `phpunit.xml.dist` file. In fact, Travis CI didn't set this environment variable :wink: